Fluent/color.ini

```diff
@@ -53,3 +57,30 @@ accent             = 00BEFD
 
 layer-shadow       = 1d1d1d
 contour            = 303030
+
+[comfy]
```

Comment:
Add a preview of this colour scheme in [THEMES.md](/morpheusthewhite/spicetify-themes/blob/master/THEMES.md)